### ISSUE TITLE
Generated token is not recognized

### ISSUE THREAD

**Issue by Adrian-Bielefeldt** [zeratax/matrix-registration#89]
**Describe the bug**
The generated token is always flagged as invalid. This means no account can be created.

Specifically, if you access https://matrix.bielefeldt.berlin/matrix-registration/register?token=VampireTobaccoAndroid and try to create an account, the token is not accepted, even though it was generated by this instance.

**To Reproduce**

Go to https://matrix.bielefeldt.berlin/matrix-registration/register?token=VampireTobaccoAndroid.
Try to register an account.

**Expected behavior**
The token should be accepted as valid and the account created.

**System:**
 - OS: Debian
 - python version: 3.6
 - matrix registration version [e.g. 0.5]

**Additional context**
I have not installed matrix-registration directly but via the playbook described here https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/master/docs/configuring-playbook-matrix-registration.md.

**Comment by xelantro** [comment]
First of please check if the token is correctly added in the database with:
`curl -H "Authorization: SharedSecret <APIAdminPassword>" https://matrix.bielefeldt.berlin/matrix-registration/token`

**Comment by Adrian-Bielefeldt** [comment]
The error was on my part. I just realised that the tutorial I copied the command for generating the token from set the expiration_date to 2021, which I failed to realise is already over.

I'm thinking it would probably be best to prevent generating a token that has already expired, but maybe I'm missing comlications that could arise from that (especially with regards to time zones or something).
